packages/react-components/react-motion-components-preview/stories/src/Slide/SlideCardsDemo.stories.tsx

```diff
@@ -0,0 +1,158 @@
+import * as React from 'react';
+import { makeStyles, tokens, Button, Card, CardHeader, CardPreview, Body1 } from '@fluentui/react-components';
+import { SlideRelaxed } from '@fluentui/react-motion-components-preview';
+
+const useClasses = makeStyles({
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '20px',
+    padding: '20px',
+    maxWidth: '800px',
+  },
+  controls: {
+    display: 'flex',
+    gap: '10px',
+    marginBottom: '20px',
+  },
+
+  cardGrid: {
+    display: 'grid',
+    gridTemplateColumns: 'repeat(auto-fit, minmax(200px, 1fr))',
+    gap: '15px',
+    alignItems: 'start',
+  },
+  card: {
+    height: '150px',
+  },
+  cardPreview: {
+    height: '80px',
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    fontSize: tokens.fontSizeBase600,
+  },
+});
+
+const cardData = [
+  {
+    id: 1,
+    title: 'Welcome',
+    content: 'Getting started',
+    color: tokens.colorPaletteBlueBorderActive,
+    fromX: '-50px',
+    fromY: '0px',
+  },
+  {
+    id: 2,
+    title: 'Features',
+    content: 'Explore the capabilities',
+    color: tokens.colorPaletteGreenBorderActive,
+    fromX: '0px',
+    fromY: '-50px',
+  },
+  {
+    id: 3,
+    title: 'Customize',
+    content: 'Make it your own',
+    color: tokens.colorPalettePurpleBorderActive,
+    fromX: '50px',
+    fromY: '0px',
+  },
+  {
+    id: 4,
+    title: 'Advanced',
+    content: 'Pro techniques',
+    color: tokens.colorPaletteRedBorderActive,
+    fromX: '-30px',
+    fromY: '30px',
+  },
+  {
+    id: 5,
+    title: 'Examples',
+    content: 'Real-world usage',
+    color: tokens.colorPaletteYellowBorderActive,
+    fromX: '0px',
+    fromY: '50px',
+  },
+  {
+    id: 6,
+    title: 'Resources',
+    content: 'Documentation & help',
+    color: tokens.colorPaletteBrassBackground2,
```

Comment:
nit: plz use `foreground` & `background` paring i.e. `colorPaletteBlueForeground2` + `colorPaletteBlueBackground2` otherwise there is no enough contrast in some examples:

<img width="236" alt="image" src="https://github.com/user-attachments/assets/9f189acf-88a5-41b5-aec6-7d6742550d00" />

Reply:
Done.